main.py

```diff
@@ -1,9 +1,185 @@
-print("Wyświetlenie listy zrobionych zadań: 1")
-print("Wyświetlenie listy zadań do wykonania: 2")
-print("Dodaj zadanie: 3")
-while True:
-    odpowiedz = input('Wybierz menu ')
-    if odpowiedz == "3":
-        nowe_zadanie = input("Napisz jakie zadanie chcesz dodać: ")
-        print(nowe_zadanie)
-        break
\ No newline at end of file
+import sqlite3
+from sqlite3 import Error
+from task import task
+
+task_list = []
+database = r"C:\Users\mateusz.saganowski\PycharmProjects\pythonProject\pythonsqlite.db"
```

Comment:
1. Do repo nie powinny trafiać ścieżki uzależnione od Twojego środowiska - gdybym teraz pobrał ten projekt, to dostałbym błąd, bo u mnie na kompie nie będzie katalogu `mateusz.saganowski`.
2. Jeśli odnosisz się do plików, które znajdują się w katalogu z apką, to możesz po prostu samą nazwę podać
```suggestion
database = "pythonsqlite.db"
```

Reply:
Cenna uwaga, poprawiono.